tests/commands/test_api_error.py

```diff
@@ -331,13 +385,13 @@ def test_all_workflow_when_server_down(self):
         # test commands
         result = self.cli("verify")
         self.assertEqual(result.exit_code, 0)
-        # Prior to 3.6, `Response` object can't be obtained.
-        if compare_version([int(x) for x in platform.python_version().split('.')], [3, 7]) >= 0:
-            assert tracking.call_count == 2
+        self.assert_tracking_count(tracking=tracking, count=2)
```

Comment:
Could you comment on why did it send events twice?

Reply:
[7b1e00a](https://github.com/launchableinc/cli/pull/616/commits/7b1e00a79f4f5c6c61658c2bf1dee1d853c405b4)